lib/scripts.js

```diff
@@ -78,15 +73,37 @@ impress.createScript = (application, fileName, callback) => {
   }
   api.fs.readFile(fileName, (err, code) => {
     if (err) {
-      application.log.error(impress.CANT_READ_FILE + fileName);
-      callback(err);
+      if (ignoredCodes.includes(err.code)) {
+        callback(null, null);
+      } else {
+        application.log.error(impress.CANT_READ_FILE + fileName);
+        callback(err);
+      }
     } else {
       exports = prepareScript(application, fileName, code);
       callback(null, exports);
     }
   });
 };
 
+// Create script if file exists
+//   application <Application>
+//   fileName <string> absolute path
+//   callback <Function>
+//     err <Error>
+//     exports <Function> | <null>
+impress.createScriptOpt = (application, fileName, callback) =>
```

Comment:
We do not need additional function for optional scripts. Better to handle err-back at the higher level.

Reply:
But this way the error will be written to logs as `error`. IMO it will be very confusing.
Is that okay?

Reply:
We need to move log writing higher too